CONTRIBUTING.md

```diff
@@ -82,17 +130,16 @@ Must be one of the following:
 - **feat**: A new feature
 - **fix**: A bug fix
 - **docs**: Documentation only changes
-- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing
-  semicolons, etc.)
+- **style**: Changes that do not affect the meaning of the code (white-space, formatting, missing semicolons, etc.)
 - **refactor**: A code change that neither fixes a bug nor adds a feature
 - **perf**: A code change that improves performance
 - **test**: Adding missing or correcting existing tests
-- **chore**: Changes to the build process or auxiliary tools and libraries such as documentation
-  generation
+- **chore**: Changes to the build process or auxiliary tools and libraries such as documentation generation
+- **types**: A code change related with TypeScript type
 
 ### Scope
 
-The scope could be anything specifying place of the commit change. For example `useQuery`, `useMutation` etc...
+The scope could be anything specifying place of the commit change. For example `query-core`, `react-query` etc...
```

Comment:
TanStack Query has become a framework-independent library, and this monorepo is developing adapters for TanStack Query for various frameworks.

So I thought it would be better for this scope to use the library name (react-query) rather than a name that could be duplicated, such as useQuery.

https://github.com/TanStack/query/commits/main/

![image](https://github.com/TanStack/query/assets/61593290/4f5a71ad-1060-4384-8f52-74b4639d9e0a)